tools/android/build_aar.py

```diff
@@ -145,7 +145,7 @@ def path_from_env_var(env_var: str):
     parser.add_argument(
         "--api-level",
         type=int,
-        default=24,
+        default=21,
         help="Android API Level. E.g., 24.",
```

Comment:
why do we lower to 21? 
nit: update the help.

Reply:
Mentioned in the review description. ORT uses 21. If you attempt to use both the ORT and extensions packages the build complains about incompatibility.